app/src/main/java/test/arctouch/tmdbapp/data/TMDbDataSource.java

```diff
@@ -0,0 +1,12 @@
+package test.arctouch.tmdbapp.data;
+
+import rx.Observable;
+import test.arctouch.tmdbapp.network.model.MovieResponse;
+
+public interface TMDbDataSource {
```

Comment:
I didn't do because I like to use the approach to test the presenter. For example, on my test class I could just implement a mock repository, although I didn't have time to implement the tests. I also like to follow the SOLID principles where we need to depend on the interface not the implementation.